src/Orchard.Web/Modules/Orchard.OpenId/Views/EditorTemplates/Parts.AzureActiveDirectorySettings.cshtml

```diff
@@ -59,5 +59,8 @@
     @Html.LabelFor(m => m.GraphApiUrl, T("Graph API URL"))
     @Html.TextBoxFor(m => m.GraphApiUrl, new { @class = "text large" })
     <span class="hint">@T("Typically https://graph.windows.net")</span>
+
+    @Html.LabelFor(m => m.GraphApiKey, T("Graph API Key"))
```

Comment:
I will merge the PR, but can you create a new one that removes the usage of LabelFor and instead use a `<label>` tag? Otherwise there can be issues with double-encoding (try to use an accent for instance, or `<` in the text)